### ISSUE TITLE
Undefined array key "REQUEST_METHOD"

### ISSUE THREAD

**Issue by saddy001** [ahmedkaludi/wp-multilang#88]
In the logs (most recent plugin version):

`Undefined array key "REQUEST_METHOD" in /var/www/html/test/wp-content/plugins/wp-multilang/includes/wpm-core-functions.php:225`

**Comment by saddy001** [comment]
Fix:
```
/var/www/html/test/wp-content/plugins/wp-multilang/includes/wpm-core-functions.php:225

if ( 'GET' === ($_SERVER['REQUEST_METHOD'] ?? null) || ! is_admin() ) {
```

**Comment by shridhamdeveloper** [comment]
This issue already resolve in below commit

https://github.com/ahmedkaludi/wp-multilang/commit/9b9428d27622e5fd839d05fec27d834aa962ac1a

**Comment by Sanjeevsetu** [comment]
@saddy001 

It has been fixed and released in previous version 2.4.12. I am closing it.
